docs/cumulusci_flow.rst

```diff
@@ -0,0 +1,273 @@
+         prefix_feature: feature/
+         prefix_beta: beta/
+         prefix_release: release/
+
+These values can be changed to match naming conventions used by your own project.
+
+
+
+.. _auto-merged:
+
+Auto Merging
+------------
+CumulusCI helps to keep large diffs and merge conflicts from being the norm. CumulusCI's auto-merge functionality helps teams:
+
+* Keep feature branches up-to-date with the ``main`` branch (main to feature merges)
+* Manage long-lived feature branches for larger features worked on by multiple developers (parent to child merges)
+* Manage large releases that occur several times a year (release to future release merges).  
+
+
+Main to Feature Merges 
+^^^^^^^^^^^^^^^^^^^^^^
+One of the bigger differences between CumulusCI Flow and Github Flow or git-flow is that we automate the merging of commits to a projects main branch into all open feature branches.  This auto-merge does a lot for us:
+
+* Ensures feature branches are in sync with the  main branch.
+* Re-tests each feature branch with any changes to main since the merge generates a new commit
+* Eliminates merge conflicts when merging a Pull Request to main
+
+To understand the benefit of auto-merging to feature branches, consider the following scenario: A developer starts work on a feature branch, puts in a few weeks on it, and then has to leave unexpectedly for a few months.  While they are on leave, their feature branch gets automatically updated with any new commits on main and rebuilt.  A few weeks into their leave, a new commit on main gets merged to their feature branch and breaks the build.  When the developer returns after their leave, they can look at the build history to find which commit from main broke their feature branch.
+
+Without auto-merging, the developer would return, merge main into their feature branch, and then have to sift through all the commits to main during their leave to figure out which one broke their feature branch.  More testing and build history is always a good thing in addition to the other benefits we gain from auto-merging.
+
+CumulusCI facilitates the auto-merge to feature branches via the ``github_automerge_main`` task which is included by default in the ``release_beta`` flow.
+
+
+
+Parent to Child Merges
+^^^^^^^^^^^^^^^^^^^^^^
+As we've worked in the CumulusCI Flow for the last 4+ years, we've occasionally seen the need for longer running, collaborative feature branches that are used by multiple developers to work on different parts of a single large feature. The solution was to expand the concept of auto-merging main-to-feature branches to also handle the concept of Parent and Child Feature Branches.
+
+Parent/Child Feature Branches are created using a simple naming format for branches:
+
+* **Parent**: feature/parent-branch-name
+* **Child**: feature/parent-branch-name__child-branch-name
+
+If this combination of named parent and child branches exist, the auto-merging functionality changes a bit:
```

Comment:
Is this really a change? It seems more like just an elaboration of the automerge we've talked about.